routes.php

```diff
@@ -11,4 +11,6 @@
 
 Route::get('sitemap.xml', [SitemapController::class, 'index']);
 
-Route::get('favicon.ico', [FaviconController::class, 'index']);
+Route::get('favicon', [FaviconController::class, 'index']);
```

Comment:
I'm worried about the `favicon.ico` routing. I think it's a better way to leave `favicon.ico` and adding `favicon` and both to point the same method in the same controller. Even only for backward compatibility.